io.github.kvazar_network.kevacoin.json

```diff
@@ -0,0 +1,157 @@
+                "./configure --prefix=/app && make -j $FLATPAK_BUILDER_N_JOBS && make install"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.libsodium.org/libsodium/releases/libsodium-1.0.20.tar.gz",
+                    "sha256": "ebb65ef6ca439333c2bb41a0c1990587288da07f6c7fd07cb3a18cc18d30ce19"
+                }
+            ]
+        },
+        {
+            "name": "libprotobuf",
+            "buildsystem": "simple",
+            "build-commands": [
+                "./autogen.sh",
+                "./configure --prefix=/app",
+                "make -j $FLATPAK_BUILDER_N_JOBS && make install"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/protocolbuffers/protobuf/releases/download/v2.6.0/protobuf-2.6.0.tar.gz",
+                    "sha256": "cd087148a8651aaaf49e6e47df9491f3cd4cc58d3add0ac7d7b834ad808f4f13"
+                }
+            ]
+        },
+        {
+            "name": "libevent",
+            "buildsystem": "simple",
+            "build-commands": [
+                "./autogen.sh",
+                "./configure --prefix=/app",
+                "make -j $FLATPAK_BUILDER_N_JOBS && make install"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/libevent/libevent/archive/refs/tags/release-2.1.12-stable.tar.gz",
+                    "sha256": "7180a979aaa7000e1264da484f712d403fcf7679b1e9212c4e3d09f5c93efc24"
+                }
+            ]
+        },
+        {
+            "name": "libzeromq",
+            "buildsystem": "simple",
+            "build-commands": [
+                "./autogen.sh",
+                "./configure --prefix=/app",
+                "make -j $FLATPAK_BUILDER_N_JOBS && make install"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/zeromq/libzmq/releases/download/v4.3.5/zeromq-4.3.5.tar.gz",
+                    "sha256": "6653ef5910f17954861fe72332e68b03ca6e4d9c7160eb3a8de5a5a913bfab43"
+                }
+            ]
+        },
+        {
+            "name": "miniupnpc",
+            "buildsystem": "simple",
+            "build-commands": [
+                "make -j $FLATPAK_BUILDER_N_JOBS && make install INSTALLPREFIX=/app"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/miniupnp/miniupnp/releases/download/miniupnpc_2_2_1/miniupnpc-2.2.1.tar.gz",
+                    "sha256": "3a3167e57727bf1d2a7b4861f7c7b57a663f58b9cf68227762ed2fc64e8ea11f"
+                }
+            ]
+        },
+        {
+            "name": "core",
+            "buildsystem": "simple",
+            "build-commands": [
```

Comment:
this should be `buildsystem: autotools` instead.

Reply:
You really want to oppose the reviewers? We can just close the PR if you prefer. That will save everyone's time.

Whatever you found above is wrong.